plugins/plugin-pullrequest-parent/che-plugin-pullrequest-ide/src/main/java/org/eclipse/che/plugin/pullrequest/client/steps/InitializeWorkflowContextStep.java

```diff
@@ -100,7 +100,24 @@ public void apply(final List<Remote> remotes) throws OperationException {
         };
     }
 
-    private String getContributeToBranchName(final ProjectConfig project) {
+    protected void setContributeToBranchName(Context context) {
+        String contributeToBranchName = getBranchFromProjectMetadata(context.getProject());
+
+        if (contributeToBranchName != null) {
+            context.setContributeToBranchName(contributeToBranchName);
+            return;
+        }
+
+        vcsServiceProvider.getVcsService(context.getProject()) //
```

Comment:
could you please remove ```//``` at the end of lines. It's unusual for che such kind of formatting technique.